plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1798,6 +1798,52 @@ public void testVacuumWithTrailingSlash()
         }
     }
 
+    @Test
+    public void testVacuumWithWhiteSpace()
+            throws Exception
+    {
+        String catalog = getSession().getCatalog().orElseThrow();
+        String tableName = "test_vacuum_white_space_" + randomNameSuffix();
+        String tableLocation = getLocationForTable(bucketName, tableName) + "/";
+        Session sessionWithShortRetentionUnlocked = Session.builder(getSession())
+                .setCatalogSessionProperty(catalog, "vacuum_min_retention", "0s")
+                .build();
+
+        assertUpdate(format("CREATE TABLE %s (val int, col_white_space timestamp(6)) WITH (location = '%s', partitioned_by = ARRAY['col_white_space'])", tableName, tableLocation));
+
+        assertUpdate("INSERT INTO " + tableName + " VALUES (1, TIMESTAMP '2024-12-13 11:00:00.000000'), (2, TIMESTAMP '2024-12-13 12:00:00.000000')", 2);
```

Comment:
Could you use `TestTable` so we can omit DROP TABLE statement?

Reply:
It seems the test needs to specify the tableLocation while creating the the table, the tableLocation are build with the tableName, thus seems can not use the TestTable.

Reply:
That seems a bug in the helper method. I'm fine with this style for now, but please move INSERT statement to inside of try block. Otherwise, a table won't be dropped when the statement fails for some reason.